sdk/helper/keysutil/policy.go

```diff
@@ -1449,6 +1452,14 @@ func (p *Policy) Rotate(ctx context.Context, storage logical.Storage, randReader
 		}
 	}
 
+	if p.Keys == nil {
```

Comment:
Nit: maybe do `p.LatestVersion += 1` first to keep `p.Keys` context closer to the assignment.

Reply:
Done in 0e7b526.